pulp_rpm/app/modelresource.py

```diff
@@ -409,9 +412,13 @@ def set_up_queryset(self):
             UpdateCollections belonging to UpdateRecords for a specified repo-version.
 
         """
-        return UpdateCollection.objects.filter(
-            update_record__in=UpdateRecord.objects.filter(pk__in=self.repo_version.content)
-        ).order_by("pulp_id")
+        return (
+            UpdateCollection.objects.filter(
+                update_record__in=UpdateRecord.objects.filter(pk__in=self.repo_version.content)
+            )
+            .order_by("pulp_id")
+            .select_related("update_record")
```

Comment:
Interesting - while it "makes sense" to fetch the associated advisory, was this a response to an oberved problem?

Reply:
All of it is in response to observed problems, yeah.  Specifically: look at the ForeignKeyWidget.  Rather than using the PK, it uses the digest, which means it hits up the `UpdateRecord` so that it can get the digest.

Reply:
:+1: